starport-smart-contracts/index.md

```diff
@@ -0,0 +1,213 @@
+# Smart contracts with CosmWasm
```

Comment:
could we do this in a separate PR? I think we could coordinate something with the cosmwasm team instead of having our own tutorial.